tests/run/datetime_pxd.pyx

```diff
@@ -276,6 +277,6 @@ def test_datetime_from_timestamp():
     >>> tp == dt
     True
     """
-    tp = datetime_from_timestamp(1618770890)
-    dt = py_datetime.datetime(2021, 4, 18, 20, 34, 50)
+    tp = datetime_from_timestamp(py_time.time())
+    dt = py_datetime.datetime.today()
```

Comment:
These two should probably use the same timestamp and not two different (microsecond-)times.

Reply:
`datetime.today()` is actually defined in the docs as:
> Return the current local datetime, with tzinfo None.
> 
> Equivalent to:
> ```py
> datetime.fromtimestamp(time.time())
> ```
> See also now(), fromtimestamp().
> 
> This method is functionally equivalent to now(), but without a tz parameter.

Though I agree that it could be made more apparent.